custom/conf/app.example.ini

```diff
@@ -1290,6 +1290,15 @@ ROUTER = console
 ;; Maximum allowed file size in bytes to render CSV files as table. (Set to 0 for no limit).
 ;MAX_FILE_SIZE = 524288
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[ui.emoji_aliases]
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; Set custom emoji aliases. A custom emoji alias maps a custom name to a predefined emoji name.
+;custom = emoji
```

Comment:
Can you copy the description from `config-cheat-sheet.en-us.md` into here? If I read only what is here, I'd be confused.

Reply:
> Can you copy the description from `config-cheat-sheet.en-us.md` into here? If I read only what is here, I'd be confused.

Sure, the following is copied from config-cheat-sheet.en-us.md

### UI - Custom Emoji Aliases (`ui.emoji_aliases`)

Add self-defined emoji aliases into this section, for example,
**thumbs_up = thumbsup**
will map the self-defined alias "thumbs_up" to gitea defined alias "thumbsup". Add as many mapping pairs as you need. Please note that self-defined aliases are in priority.

Reply:
Copied!

Reply:
Thanks, looks good now